compose/service.py

```diff
@@ -479,6 +488,7 @@ def build(self, no_cache=False):
             stream=True,
             rm=True,
             nocache=no_cache,
+            fileobj=self._get_fileobj(),
```

Comment:
That would be great. I think we can ask for a new release when this is ready.

I believe fileobj doesn't actually work when the Dockerfile makes use of other files in the build context.